diff --git a/python/setup.py b/python/setup.py
index 1a20eb4..0045f79 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -9,7 +9,7 @@ nn_plugin_header_dir = '@NN_PLUGIN_HEADER_DIR@'
 nn_plugin_library_dir = '@NN_PLUGIN_LIBRARY_DIR@'
 torch_dir, _ = os.path.split('@TORCH_LIBRARY@')

-extra_compile_args = ['-std=c++17']
+extra_compile_args = ['-std=c++17', '-D_GLIBCXX_USE_CXX11_ABI=1']
 extra_link_args = []
 libraries = ['OpenMM', 'OpenMMTorch']
 runtime_library_dirs = [os.path.join(openmm_dir, 'lib'), torch_dir]
