diff --git a/python/setup.py b/python/setup.py
index db6b9bb..e3490c6 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -27,9 +27,12 @@ extension = Extension(name='_openmmtorch',
                       extra_link_args=extra_link_args
                      )

+CUDA_VERSION=os.getenv('CUDA_VERSION')
+HIP_VERSION=os.getenv('HIP_VERSION')
 setup(name='openmmtorch',
       version=version,
       py_modules=['openmmtorch'],
       ext_modules=[extension],
-      install_requires=['openmm', 'torch']
+      install_requires=['openmm', 'torch'],
+      extras_require={"cuda12": [f"OpenMM-Torch-CUDA-12=={version}"], "cuda13": [f"OpenMM-Torch-CUDA-13=={version}"], "hip6": [f"OpenMM-Torch-HIP-6=={version}"], "hip7": [f"OpenMM-Torch-HIP-7=={version}"]}
      )
